Sources/CoreToolkit/Optional+orThrow.swift

```diff
@@ -0,0 +1,12 @@
+public enum OptionalError: Error {
```

Comment:
To je overkill mit enum, kterej ma jeden case ne? Ja bych to udelal jako prazdnou strukturu, ktera se proste bude jmenovat `OptionalError`.

Reply:
Nebo at se jmenuje podle toho case...